test/hotspot/jtreg/containers/docker/TestCPUAwareness.java

```diff
@@ -201,13 +218,15 @@ private static void testAPCCombo(String cpuset, int quota, int period, int share
             .addDockerOpts("--cpu-quota=" + quota)
             .addDockerOpts("--cpu-shares=" + shares);
 
+        if (useContainerCpuShares) opts.addJavaOpts("-XX:+UseContainerCpuShares");
```

Comment:
The JDK 18u patch has an additional `System.out.println("useContainerCpuShares = " + useContainerCpuShares);` on line 210 of this file which is missing in this backport.